steely/plugins/roomcheck.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+'''
+.room <room number>
+
+Check if a dcu room is free right now
+
+.room <GLA.LG26>
+'''
+
+
+import requests
+from datetime import datetime
+import sys
+import bs4
+
+
+__author__ = 'gruunday'
+COMMAND = '.room'
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    def send_message(message):
+        bot.sendMessage(message, thread_id=thread_id, thread_type=thread_type)
+    if not message:
+        send_message('.room <room number>\nCheck if a dcu room is free right now\n.room <GLA.LG26>')
+        return
+    booking = check_room(message)
+    if booking:
+        send_message(message + ' is not free, there is\n \n' + booking + '\n')
+    else:
+        send_message(message + " is currently free\n")
+
+
+def check_room(room):
+    url = get_url(room)
+    # Downloads request from url URL and stores as string in res
+    res = requests.get(url)
+    # Creates Beautiful Soup out of html
+    current_soup = bs4.BeautifulSoup(res.text, "lxml")
+    # Takes all <tr> tags
+    elems = current_soup.select('tr')
+    return elems[14].getText().strip()
```

Comment:
Pedantic comment from a comment-disliker: These comments are a little repetitive of the source code.